src/common/post.ts

```diff
@@ -184,6 +187,20 @@ export const createFreeformPost = async (
       private: privacy,
     },
   });
+
+  const vordrStatus = await checkWithVordr(
+    { post: createdPost },
+    { con, userId: ctx.userId, req: ctx.req },
+  );
+
+  if (vordrStatus === true) {
+    createdPost.banned = true;
+    createdPost.showOnFeed = false;
```

Comment:
I think @capJavert was right we also need to set it as flags.
Actually Ante, not sure now which one we use, maybe we should only use either raw or the flags?

Reply:
I set both column of flags now